source/install/ config-ssl-http2-apache2.rst

```diff
@@ -0,0 +1,60 @@
+.. _config-ssl-http2-apache2:
+
+Configuring Apache2 with SSL and HTTP/2 (Unofficial) 
+=====================================
+
+Once you've configured Apache2 as a proxy for your Mattermost Server, the easiest way to enable SSL on Apache2 is via Let's Encrypt and [Certbot](https://certbot.eff.org/#ubuntuxenial-apache). Once installed, simply run:
+``$ certbot --apache``
+And follow the straight forward guide. Afterwards you should find a new configuration file in ``/etc/apache2/sites-available`` which should follow the format: ``mysubdomain.mydomain.com-le-ssl.conf``.
+
+Once opened it should look something like the following:
+
+.. code-block:: none
+	<IfModule mod_ssl.c>
```

Comment:
You need a space between the code-block directive and the next line, like this:
```
.. code-block:: none

    <IfModule mod_ssl.c>
```
otherwise the code doesn't show up in the output.